tools/build_defs/cmake_script.bzl

```diff
@@ -73,6 +77,10 @@ def create_cmake_script(
     if not params.cache.get("CMAKE_RANLIB"):
         params.cache.update({"CMAKE_RANLIB": ""})
 
+    if len(make_path.split('/')) != 1:
```

Comment:
what is this accounting for?

Reply:
This is for the case that its using a builtin tool that isn't linked into the sandbox e.g. the system installed `cmake`